packages/react-storage/src/components/StorageBrowser/components/Table.tsx

```diff
@@ -29,50 +28,48 @@ interface TableProps {
 
 export const Table = ({ headers, rows }: TableProps): React.JSX.Element => {
   return (
-    <ViewElement className={`${STORAGE_BROWSER_BLOCK}__data-table`}>
```

Comment:
Removed this element so that the View that renders the data table can control it so the loader and messages can appear inside the table

Reply:
I could've sworn @jordanvn had removed this earlier